src/components/accessories/dateField/DateField.tsx

```diff
@@ -25,7 +26,15 @@ const DateField: FunctionComponent<IProps> = ({
   const [value, setValue] = useState<Date | null>(null);
 
   useEffect(() => {
-    setValue(fieldValue === "" ? null : new Date(+fieldValue));
+    if (fieldValue === "") {
+      setValue(null);
+    } else if (moment(+fieldValue).isValid()) {
+      setValue(moment(+fieldValue).toDate());
+    } else if (moment(fieldValue, "DD/MM/YYYY").isValid()) {
+      setValue(moment(fieldValue, "DD/MM/YYYY").toDate());
+    } else {
+      setValue(moment(fieldValue).toDate());
+    }
```

Comment:
Here man sorry but I can't understand what is the purpose of this.

Reply:
The purpose is to fill  the date field with the fieldValue prop. The problem here is the format. This old code `
setValue(fieldValue === "" ? null : new Date(+fieldValue));` is not working for edit case. For example when you pass "30/08/1991" in edit patient mock data, it does'nt work. So we have to convert data fieldValue before setting.

Reply:
The mocked value should be the same as the patientDTO, the date should comes from the server as a timestamp string.

The dateField component must be agnostic than the date format used.

Reply:
OK. the date used in cypress mock for edit patient was   "birthDate": "30/08/1991", So I will also replace that to timestamp.

Reply:
I think at least we have to consider two cases for date: timestamp and millisecondes date. So I update the dates converters according to that two possibilities.